deb-package-builder/extensions/cassandra-cpp-driver/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+source ${DEB_BUILDER_DIR}/extensions/functions.sh
+
+echo "Building cassandra cpp driver"
+
+echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
```

Comment:
Isn't it safe to do this in the `Dockerfile`? It may affect other builds depending the execution order.